spec/helpers/notifications_helper_spec.rb

```diff
@@ -38,13 +38,13 @@
         @note = double()
       end
 
-      it 'with two, does not comma seperate two actors' do
+      it 'with two, does not comma separate two actors' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.